src/serialization/serialization.rs

```diff
@@ -176,13 +183,19 @@ mod benchmarks;
 mod v2_serializer;
 pub use self::v2_serializer::{V2Serializer, V2SerializeError};
 
+#[path = "v2_deflate_serializer.rs"]
```

Comment:
You know that Rust, when confronted with `mod foo`, will look for both `foo/mod.rs` *and* `./foo.rs` by default, right? I don't think this `#[path]` should be necessary.